### PR TITLE
Weekly digest

### PR DESCRIPTION
**Pull request by dshevtsov**
## This PR is a:

News update

## Summary

When this pull request is merged, it will add news updates for the passed week.

### CODE REVIEW

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,52 @@ description: |
...
-updated: Wed Mar 27 18:03:03 2019
+updated: Tue Apr  2 11:41:12 2019
 entries:
+- description: Added a topic about [how to retrieve](https://devdocs.magento.com/guides/v2.3/graphql/get-customer-authorization-token.html)
+    the customer's authorization token for GraphQL queries and mutations.
+  versions: 2.3.1
+  type: New topic
+  date: April  1, 2019
+  link: https://github.com/magento/devdocs/pull/4052
+- description: Added section about mutations in [Resolvers](https://devdocs.magento.com/guides/v2.3/graphql/develop/resolvers.html)
```
Comment (shrielenee) [inline _data/whats-new.yml]: The previous item's sentence has ending punctuation but this one does not. I advise reconciling punctuation.

File: _data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,52 @@ description: |
...
+  type: Major update
+  date: March 29, 2019
+  link: https://github.com/magento/devdocs/pull/4087
+- description: Moved the MFTF docs from `<devdocs root>/mftf/2.3/` to [`<devdocs root>/mftf/docs/`](https://devdocs.magento.com/mftf/docs/introduction.html).<br/>The
+    docs have been moved to the [MFTF repository](https://github.com/magento/magento2-functional-testing-framework).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: March 28, 2019
+  link: https://github.com/magento/devdocs/pull/3898
+- description: 'Updated the [Cloud Release notes for ece-tools version 2002.0.17](https://devdocs.magento.com/guides/v2.3/cloud/release-notes/cloud-tools.html#v2002017).<br/>-
```
Comment (shrielenee) [inline _data/whats-new.yml]: This tick at the beginning of "Updated" seems unneeded.

File: _data/whats-new.yml (hunk 1 above)
Comment (dshevtsov) [reply _data/whats-new.yml]: @erikmarr please consider the comment above for #4052

File: _data/whats-new.yml (hunk 2 above)
Comment (dshevtsov) [reply _data/whats-new.yml]: It is generated by the native Ruby YAML library, and thus is trusted to be valid.

File: _data/whats-new.yml (hunk 1 above)
Comment (dshevtsov) [reply _data/whats-new.yml]: @keharper please consider to apply the above comment for #4096

Comment (erikmarr) [reply _data/whats-new.yml]: @dshevtsov PR 4052 corresponds to the description in line 9 which contains an ending period. It looks like we're fairly consistent with periods for what's new entries:
https://devdocs.magento.com/whats-new.html
My vote for reconciliation is to use periods. :-)

Comment (dshevtsov) [reply _data/whats-new.yml]: @eriksmarr, yes I removed the comment that mentioned your PR.
Sorry for confusion.

Comment (dshevtsov) [reply _data/whats-new.yml]: @shrielenee reconciled

**Comment by dshevtsov** [comment]
running tests

**Pull request merged**